### PR TITLE
Add denpendencies in Compute to resolve the CVE

### PR DESCRIPTION
**Pull request by NoriZC**
## Description
This is a pr for resolve CVE. Needs to merge soon for running daily build pipeline on main. We are experimenting with the options to resolve a security issue / an outdated dependency. After testing will revert the change and do updates waiting for Compute team's review. 
We will make sure the code is clean before the release in May.

## Checklist

- [x] Check this box to confirm: **I have read the [_Submitting Changes_](../blob/main/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/main/CONTRIBUTING.md) and reviewed the following information:**

* **SHOULD** select appropriate branch. Cmdlets from Autorest.PowerShell should go to [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch. 
* **SHOULD** make the title of PR clear and informative, and in the present imperative tense. 
* **SHOULD** update `ChangeLog.md` file(s) appropriately
    * For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    * A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header in the past tense. Add changelog in description section if PR goes into [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch.
    * Should **not** change `ChangeLog.md` if no new release is required, such as fixing test case only.
* **SHOULD** have approved design review for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) ([_Microsoft internal only_](../blob/main/CONTRIBUTING.md#onboarding)) with following situations
    * Create new module from scratch
    * Create new resource types which are not easy to conform to [Azure PowerShell Design Guidelines](../blob/main/documentation/development-docs/design-guidelines)
    * Create new resource type which name doesn't use module name as prefix
    * Have design question before implementation
* **SHOULD** regenerate markdown help files if there is cmdlet API change. [Instruction](../blob/main/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)
* **SHOULD** have proper test coverage for changes in pull request.
* **SHOULD NOT** introduce [breaking changes](../blob/main/documentation/breaking-changes/breaking-changes-definition.md) in Az minor release except preview version.
* **SHOULD NOT** adjust version of module manually in pull request

### CODE REVIEW

File: src/Compute/Compute/Compute.csproj [hunk 1]
```diff
@@ -16,12 +16,12 @@
     <PackageReference Include="Microsoft.Azure.PowerShell.AutoMapper" Version="6.2.2" />
     <PackageReference Include="Microsoft.Azure.Management.ResourceGraph" Version="2.1.0" />
     <PackageReference Include="Microsoft.Azure.Management.Compute" Version="60.0.0" />
-    <PackageReference Include="System.Security.Permissions" Version="4.5.0" />
+    <PackageReference Include="System.Security.Permissions" Version="4.7.0" />
```
Comment (isra-fel) [inline src/Compute/Compute/Compute.csproj:19]: Is this package referenced by other projects? We should update them as well if so.

Comment (NoriZC) [reply src/Compute/Compute/Compute.csproj:19]: Yes, it appears in another 8 projects including Az.Accounts. I will check them one by one. Thanks for reminding

Comment (isra-fel) [reply src/Compute/Compute/Compute.csproj:19]: Sorry I just noticed `System.Security.Permissions` is a common dependency, we should follow the similar process of [updating Azure.Core](https://eng.ms/docs/cloud-ai-platform/azure-core/azure-management-and-platforms/control-plane-bburns/azure-cli-tools-azure-cli-powershell-and-terraform/azure-cli-tools/teams_docs/azps_docs/repo_maintenance/update_azure_core#process-of-updating-azurecore)

Comment (NoriZC) [reply src/Compute/Compute/Compute.csproj:19]: I have updated all the files, if this works I will update related changelogs in projects where updated dlls are included.

**Review by isra-fel — APPROVED** [review]

**Pull request merged**